src/main/java/seedu/duke/commands/FilterCommand.java

```diff
@@ -118,7 +135,18 @@ public ArrayList<Item> getFilteredItems() {
         ArrayList<Item> filteredItems = new ArrayList<>();
         switch (filterType) {
         case "f/category":
-            filteredItems = filterCategory(filterValue);
+
+            try {
+                filterCategory(filterValue); //filterValue = "list"/"table"/"c/[keyword(s)]"
```

Comment:
Commented code here, is it legacy code? maybe we should consider removing them soon since we are reaching v2.1 release 👍🏻